raster/r.stats/stats.c

```diff
@@ -27,7 +27,8 @@ struct Node {
 
 static struct Node **hashtable;
 static struct Node *node_list = NULL;
-static int node_count = 0, total_count = 0;
+static int node_count = 0;
+static long total_count = 0;
```

Comment:
Sometimes (Windows?), `int` might be just as large as `long`. While the minimums are 16 and 32, sizes can be 32 and 32 or 32 and 64 according to https://en.cppreference.com/w/cpp/language/types (C++, C does not have that nice description but since it depends on the compiler anyway, I guess it applies the same way).

I think someone brought this up in the past, but I don't recall if we have a best practice for that.

Reply:
If `total_count` is guaranteed to be >=0 `unsigned int` could be a better alternative.

Reply:
I chose `long` because `Node.count` is `long`.

Reply:
In _r.in.pdal_, I used typedef which goes for `unsigned long` or `unsigned long long` for number of points depending on `HAVE_LONG_LONG_INT`. However, I think `grass_int64` is better as it is aligned with the new(ish) trend of fixed with integers ([C99](https://en.cppreference.com/w/c/types/integer), [C++11](https://en.cppreference.com/w/cpp/types/integer)). Signed versus unsigned and `int64_t` versus `grass_int64`, that's whole another discussion.

However, if int->long helps in your case and it fits with the other numbers there, I'm not against merging this as is.